consoleApp/src/main/java/com/infoshare/domain/NeedRequest.java

```diff
@@ -3,13 +3,15 @@
 import java.util.Date;
 import java.util.Objects;
 
+
 public class NeedRequest {
     private TypeOfHelp typeOfHelp;
     private HelpStatuses helpStatus;
     private Date statusChange;
     private PersonInNeed personInNeed;
```

Comment:
https://jira.is-academy.pl/browse/JJDZR2CL-31
czy dodanie ID do NeedRequest chcesz robić w oddzielnym pull request/ branchu?

Reply:
wszystko poprawilem na aktualnym branchu